docs/aurora/data-management/daos/daos-overview.md

```diff
@@ -164,8 +164,8 @@ To optimize performance, you may need to copy the contents of `launch-dfuse.sh`,
 
 If you get an error trying to access your container (such as on the dfuse container mount) your container may have a status of 'UNCLEAN'.  You can check this with the following command:
 
-```bash linenums="1"
-DAOS cont get-prop <pool name> <container name>
+```bash
+daos cont get-prop <pool name> <container name>
```

Comment:
I understand why you are reverting the CLI to lowercase, but why get rid of the line numbers?